system/dynomite/templates/deployment.yaml

```diff
@@ -17,6 +17,9 @@ spec:
         checksum/dynomite-config: {{ tuple $ $member $index | include "dynomite/templates/etc/_dynomite.yaml.tpl" | sha256sum }}
         checksum/redis-config: {{ tuple $ $member $index | include "dynomite/templates/etc/_redis.conf.tpl" | sha256sum }}
         checksum/bin-config: {{ include "dynomite/templates/configmap-bin.yaml" $ | sha256sum }}
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9682"
+        prometheus.io/targets: "openstack"
```

Comment:
please use the `kubernetes` prometheus as it runs in scale-out

Reply:
Updated